grafonnet/gauge.libsonnet

```
@@ -3,6 +3,8 @@
     title,
     datasource=null,
     calc='mean',
+    timeFrom=null,
```

```suggestion
    time_from=null,
```
Convention so far (for the most part) has been to use snakes instead of camels for arguments.

---

OK, singlestat is using `timeFrom`... I guess I looked at the wrong file. Will fix.

---

Apologies for the nitpick. I'm realizing perhaps the convention is instead to be wildly inconsistent in terms of case style 😅. Hoping to get this a bit more under control.